src/ray/raylet/local_object_manager.cc

```diff
@@ -239,9 +240,13 @@ void LocalObjectManager::SpillObjectsInternal(
                   pinned_objects_.emplace(object_id, std::move(it->second));
                   objects_pending_spill_.erase(it);
                 }
-
-                RAY_LOG(ERROR) << "Failed to send object spilling request: "
-                               << status.ToString();
+                std::ostringstream object_id_stream;
+                std::copy(objects_to_spill.begin(), objects_to_spill.end() - 1,
+                          std::ostream_iterator<ObjectID>(object_id_stream, ", "));
+                object_id_stream << objects_to_spill.back();
+                RAY_LOG(ERROR) << "Failed to send object spilling request for objects "
```

Comment:
Actually instead of this error message, why don't we just tell users which log files to look at? (the io worker log files that tried spilling objects).

Reply:
I added the IO worker ID to the log message for that reason, are you talking about adding a more explicit directive for the user to go look at those logs, for users that don't know that those logs exist or that the worker ID is included in the log file name?

Reply:
Yes! because we are the only people who know how to debug when there is a worker id.